joss_paper/pyfixest_joss.md

```diff
@@ -0,0 +1,64 @@
+---
+title: 'PyFixest: A Python Port of fixest for High-Dimensional Fixed-Effects Regression'
+date: 8 August 2025
+author:
+  - name: Alexander Fischer
+    affiliation: Trivago
+---
+
+
+## Summary
+
+PyFixest is an open-source Python library that implements efficient routines for regression analysis with multiple potentially high-dimensional fixed effects by applying the Frisch-Waugh-Lovell theorem. It is a faithful port of the R package fixest [@berge2018], aiming to replicate fixest’s core design principles and functionality within the Python ecosystem. Users familiar with fixest in R can seamlessly transition their analysis to Python using the same syntax and obtaining identical results. Likewise, users of PyFixest should find it easy to port their analysis to R and use the fixest package.
```

Comment:
Is there anything else PyFixEst does that fixest does not? Or could we mention the scale we support (say, to run on GPU). I think is important to emphasize it's a port but also because of a reason: the scale and how these models can integrate better in production-like environments (I know, I know you can deploy R, but all the data pipelines are in Python, so it's definitely an advantage)